src/supernode/DAPI_RPC_Client.h

```diff
@@ -66,18 +66,18 @@ namespace supernode {
 
 	    	WasConnected = false;
 	    	if(!invoke(rpc_command::DAPI_URI, rpc_command::DAPI_METHOD, req_param, timeout, std::addressof(pri))) {
-	    		LOG_PRINT_L5("Failed to invoke http request to  " << call<<"  URI: "<<m_URI);
+                LOG_PRINT_L4("Failed to invoke http request to  " << call<<"  URI: "<<m_URI);
```

Comment:
Maybe it's better to log errors to LOG_ERROR?

Reply:
Fixed